src/components/organisms/PlaceOrder/PlaceOrder.tsx

```diff
@@ -208,8 +209,14 @@ export const PlaceOrder = ({
                     dispatch({ type: 'next' });
                 }
             } catch (e) {
-                if (e instanceof Error) {
-                    setErrorMessage(e.message);
```

Comment:
For unexpected errors, this logic is needed. In the current logic, only `ContractFunctionRevertedError` can set the error message.